addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -694,45 +598,6 @@ def test_invoice_grouped_taxes_with_tax_group(self):
             'subtotals_order': ["Untaxed Amount", "Tax withholding"],
         })
 
-    def test_taxtotals_with_different_tax_rounding_methods(self):
-
-        def run_case(rounding_line, lines, expected_tax_group_amounts):
-            self.env.company.tax_calculation_rounding_method = rounding_line
-
-            document = self._create_document_for_tax_totals_test(lines)
-            tax_amounts = document.tax_totals['groups_by_subtotal']['Untaxed Amount']
-
-            if len(expected_tax_group_amounts) != len(tax_amounts):
-                self.fail("Wrong number of values to compare.")
-
-            for tax_amount, expected in zip(tax_amounts, expected_tax_group_amounts):
-                actual = tax_amount['tax_group_amount']
-                if document.currency_id.compare_amounts(actual, expected) != 0:
-                    self.fail(f'{document.currency_id.round(actual)} != {expected}')
-
-        # one line, two taxes
-        lines = [
-            (100.41, self.tax_16 + self.tax_53),
-        ]
-        run_case('round_per_line', lines, [69.29])
-        run_case('round_globally', lines, [69.29])
-
-        # two lines, different taxes
-        lines = [
-            (50.4, self.tax_17a),
-            (47.21, self.tax_17b),
-        ]
-        run_case('round_per_line', lines, [16.60])
-        run_case('round_globally', lines, [16.60])
-
-        # two lines, same tax
-        lines = [
-            (50.4, self.tax_17a),
-            (47.21, self.tax_17a),
-        ]
-        run_case('round_per_line', lines, [16.60])
-        run_case('round_globally', lines, [16.59])
-
```

Comment:
@qdp-odoo The round_per_line/globally is largely tested on test_tax.py. Since the tax_totals aggregates only the tax amounts by a certain grouping key, this test is redundant. Also, it doesn't use the default assert method to assert the whole tax totals but only some parts so... I removed it